config/core/deployments/activator-hpa.yaml

```diff
@@ -50,7 +50,7 @@ metadata:
     app.kubernetes.io/name: knative-serving
     app.kubernetes.io/version: devel
 spec:
-  minAvailable: 80%
+  minAvailable: 1
```

Comment:
I'm not convinced this is the right thing to do.

If I have 10 activators handling traffic this will now allow us to drop to only a single instance

Reply:
@dprotaso Do you have any update please?

Reply:
I don't think the point is to keep alive a percentage, but to succeed only if certain percentage is healthy which is two different things. If someone is setting X number of activators the assumption is that at least 80% should be healthy. In general in distributed systems the replicas serve not only as HA but also as a means to ensure split brain is not occurring and that there is a consensus.

Reply:
I understand your point of view but please take into consideration that 80% value is not suitable for 3 replicas since it does not allow 1 pod disruption. To allow 1 disruption, the max value is 65% (1 pod / 3rep is 33%). Also we keep a portion of alive pods.